tests/integration-tests/tests/performance_tests/resources/results/baseline/statistics.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
out of curiosity: where these values come from?

Reply:
These are the results produced by the performance test itself using a cluster config considered as a baseline, that is a cluster with the same configuration in `tests/integration-tests/tests/performance_tests/test_simple/test_simple/pcluster.config.yaml`